dynamiqs/time_tensor.py

```diff
@@ -21,6 +28,18 @@ def totime(
     dtype = dtype or get_cdtype(dtype)  # assume complex by default
     device = to_device(device)
 
+    # pwc time tensor
+    if isinstance(x, tuple) and len(x) == 3:
+        times, values, tensor = x
```

Comment:
Can you also check that `times`, `values` and `tensor` have the proper type and shape, and otherwise raise an error to explain the required shapes?